test/services/serviceGetById.spec.js

```diff
@@ -1,67 +1,83 @@
-import * as constants from '../../helpers/constants'
-import * as serviceHelper from '../../helpers/serviceHelper'
-import * as vendorHelper from '../../helpers/vendorHelper'
-import {expect} from 'chai'
+import * as constants from '../../helpers/constants';
+import * as serviceHelper from '../../helpers/serviceHelper';
+import * as vendorHelper from '../../helpers/vendorHelper';
+import { expect } from 'chai';
 const chance = require('chance').Chance();
 describe('A user should get an information about particular service', () => {
-
-    let resService, resServiceWithoutToken, resServiceWithInvalidId;
+  let res, resService, vendorId;
+  before(async () => {
+    res = await serviceHelper.getByIdService(await constants.serviceId(),process.env.TOKEN);
+    vendorId = res.body.payload.vendor['_id'];
+  });
+  after(async () => {
+    await vendorHelper.deleteVendor(vendorId);
+  });
+  describe('POSITIVE TESTS', () => {
+    it('verify response status code', () => {
+      expect(res.statusCode).to.eq(200);
+    });
+    it('verify response message', () => {
+      expect(res.body.message).to.eq('Get Service by id ok');
+    });
+    it('verify type of response message', () => {
+      expect(res.body.message).to.be.a('string');
+    });
+    it('verify type of response payload', () => {
+      expect(res.body.payload).to.be.an('object');
+    });
+    it('verify response status message', () => {
+      expect(res.res.statusMessage).to.eq('OK');
+    });
+    it('verify response payload has keys', () => {
+      expect(res.body.payload).to.have.any.keys(
```

Comment:
guess better to use `to.have.all` assertion

Reply:
I have corrected